PaddleCV/object_detection/docs/MODEL_ZOO.md

```diff
@@ -0,0 +1,81 @@
+# Model Zoo and Benchmark
+## Environment
+
+- Python 2.7.1
+- PaddlePaddle 1.5
+- CUDA 9.0
+- CUDNN 7.4
+- NCCL 2.1.2
+
+## Common settings
+
+- All models below except SSD were trained on `coco_2017_train`, and tested on the `coco_2017_val`.
+- The Batch Normalization layer of the backbone is replaced by the Affine Channel layer.
+- Unless otherwise noted, all ResNet are adopted [ResNet-B](https://arxiv.org/pdf/1812.01187) as the backbone.
+- For RCNN and RetinaNet models, only horizontal flipping data augmentation was used in the training phase and no augmentations were used in the testing phase.
+
+## Training Schedules
+
+- We adopt the exactly same training schedules as [Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules). 
+- 1x indicates the schedule starts at a LR of 0.02 and is decreased by a factor of * 0.1 after 60k and 80k iterations and finally terminates at 90k iterations for minibatch size 16. 
+- 2x indicates twice as long as the 1x schedule with the LR change points scaled proportionally.
```

Comment:
2x schedule is twice as long as 1x,  with the LR milestones scaled accordingly.

Reply:
done